superset/commands/utils.py

```diff
@@ -48,6 +49,22 @@ def populate_owners(user: User, owners_ids: Optional[List[int]] = None) -> List[
     return owners
 
 
+def populate_roles(roles_ids: Optional[List[int]] = None) -> List[Role]:
+    """
+    Helper function for commands, will fetch all roles from roles id's
+    Can raise ValidationError
```

Comment:
I believe this is what should be in the docstring:
```suggestion
    :raises RolesNotFoundValidationError: If a role in the input list is not found 
```